utils/collectd/files/exec-scripts/sqm_collectd.sh

```diff
@@ -80,6 +80,8 @@ process_qdisc() {
 
 	ifc="$1"
 	jsn=$(tc -s -j qdisc show dev "$ifc")
+	[ $? ] || return
```

Comment:
This is unreadable.  I would have used `$? -eq 0` instead.  Also... what's the urgency in getting this committed so quickly without people having a chance to comment?